app/views/govuk_publishing_components/components/_layout_for_public.html.erb

```diff
@@ -16,8 +16,6 @@
     <%= stylesheet_link_tag 'print', media: 'print' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
 
-    <!--[if lt IE 9]><%= javascript_include_tag "ie.js", integrity: true, crossorigin: "anonymous" %><![endif]-->
```

Comment:
Just wanted to check - is this no longer needed?

Reply:
Honestly not 100% sure why this was removed to begin with. 
But seeing as this is used on the live site and that I can't find any evidence that it is _not_ useful, I think the lowest risk way to proceed with this is keep it for now. It can always be removed in the future if we find no need for it.